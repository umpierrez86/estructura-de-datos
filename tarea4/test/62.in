# implementar crearBalanceado
crearBalanceado 0
crearBalanceado 1 (50, 0)
crearBalanceado 2 (40, 0) (50, 0)
crearBalanceado 3 (30, 0) (40, 0) (50, 0)
crearBalanceado 6 (10, 0) (20, 0) (30, 0) (40, 0) (50, 0) (60, 0) 
crearBalanceado 7 (10, 0) (20, 0) (30, 0) (40, 0) (50, 0) (60, 0) (70, 0) 
crearBalanceado 8 (10, 0) (20, 0) (30, 0) (40, 0) (50, 0) (60, 0) (70, 0) (1000, 0) 
crearBalanceado 15 (10, 0) (20, 0) (30, 0) (40, 0) (50, 0) (60, 0) (70, 0) (71, 0) (72, 0) (73, 0) (74, 0) (75, 0) (76, 0) (77, 0) (78, 0)
crearBalanceado 16 (10, 0) (20, 0) (30, 0) (40, 0) (50, 0) (60, 0) (70, 0) (71, 0) (72, 0) (73, 0) (74, 0) (75, 0) (76, 0) (77, 0) (78, 0) (1000, 0) 
Fin