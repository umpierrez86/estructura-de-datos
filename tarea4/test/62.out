1>#  implementar crearBalanceado.
2>
3>
(50,0.00)
4>
-(50,0.00)
(40,0.00)
5>
-(50,0.00)
(40,0.00)
-(30,0.00)
6>
--(60,0.00)
-(50,0.00)
--(40,0.00)
(30,0.00)
--(20,0.00)
-(10,0.00)
7>
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
8>
---(1000,0.00)
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
9>
---(78,0.00)
--(77,0.00)
---(76,0.00)
-(75,0.00)
---(74,0.00)
--(73,0.00)
---(72,0.00)
(71,0.00)
---(70,0.00)
--(60,0.00)
---(50,0.00)
-(40,0.00)
---(30,0.00)
--(20,0.00)
---(10,0.00)
10>
----(1000,0.00)
---(78,0.00)
--(77,0.00)
---(76,0.00)
-(75,0.00)
---(74,0.00)
--(73,0.00)
---(72,0.00)
(71,0.00)
---(70,0.00)
--(60,0.00)
---(50,0.00)
-(40,0.00)
---(30,0.00)
--(20,0.00)
---(10,0.00)
11>Fin.
